test/cli/test_one.sh

```diff
@@ -10,7 +10,7 @@ if ! diff "$expect" -u <("$script"); then
 ================================================================================
 There were differences in the captured output when running this CLI test.
 To make this output the expected output, run this and commit the changes:
-./bazel test //test/cli:$3
+./bazel test --config=dbg //test/cli:$3
```

Comment:
Is this because there is a difference in the output from  using `--config=dbg` vs the default configuration?

Or is this just for convenience, because the `--config=dbg` configuration is more typical?

E.g. I'm worried if our CLI tests _must_ be generated in dbg mode as a result of this change, because I don't think that's currently a restriction.

Reply:
Just for convenience so that build hits the cache, not needed for correctness.